BoldGrid/post-and-page-builder
# Resolves Issue - Advanced Controls panel show on Tutorial Videos when they should not

ISSUE: Resolves #603 

# Advanced Controls panel show on Tutorial Videos when they should not #

## Test Files ##

[post-and-page-builder-1.27.1-issue-603.zip](https://github.com/user-attachments/files/16996345/post-and-page-builder-1.27.1-issue-603.zip)

## NOTES TO TESTER ##
### Please leave comments regarding issues found in testing directly on the issue linked above, not the Pull Request. This helps ease the process of reviewing the testing of multiple PRs in a given project from the project view. ###
### Please remember to move this item from 'Needs Review' to either 'In Progress' or 'Awaiting Release' depending on the results of your testing ###

## Testing Instructions ##

1. Install the test zip files linked above.
2. Use Crio + PPB + Inspirations so the tutorial videos will show
3. Open a page or post
4. Click an element and then Customize Design to open the Advanced Controls
5. Close the popover
6. Click the ? icon in the DropTab and then Tutorial Videos
7. Ensure that the Advanced Controls navigation does not show on the left side of the panel.
